.editorconfig

```
@@ -24,5 +24,8 @@ indent_size = 4
 indent_style = tab
 
 [{*.bash,*.sh}]
+indent_style = space
```

Please let's use tabs. Here-documents don't work properly, when using spaces. (Therefore there is the exception for this case in some [style-guides](<https://google.github.io/styleguide/shellguide.html#s5.1-indentation>).)
```suggestion
indent_style = tab
```